src/base/settingsstorage.cpp

```diff
@@ -65,12 +65,12 @@ namespace
     QString mapKey(const QString &key)
     {
         static const MappingTable keyMapping = {
```

Comment:
Just asking. What is the idea behind a key remapping?

Because you change/update the key string?
Then I would imagine you read from both old & new key and write with new key.